### DISCUSSION TITLE
How does it work with Webstorm

### DISCUSSION THREAD

**Discussion by vjau** [fwouts/previewjs#439] [Q&A] [answered]
Hi,
i have tried to make this work with Webstorm, but nothing seems to happen.
I have installed the plugin, i go trought jsx or tsx file, but i don't see anything special, no popup, no menu.
Am i missing something ?
Webstorm 2001.3.3

**Comment by fwouts** [comment]
Hi @vjau, sorry to hear! Do you see a "Preview.js logs" bottom panel at all? What OS are you using?

**Reply by vjau** [reply]
No panel at all. The only proof that the plugin is installed is when looking in the plugin dialog. 
I'm on Ubuntu 20.04.4 LTS

**Reply by fwouts** [reply]
Hi @vjau, I installed Ubuntu 20.04 on a virtual machine and can confirm that it works for me, but it might be something to do with your project.

Could you try cloning https://github.com/fwouts/hungry, running `yarn install` and opening `design/HeroHeader/HeroHeader.tsx` to see if Preview.js works with that project?

**Reply by vjau** [reply]
Sadly, it doesn't work either with this repo.

**Reply by fwouts** [reply]
Could you share what version of `node` and `npm` you use?

**Reply by vjau** [reply]
I have found something, for some reason, node remote interpreter plugin was disabled. When enabling it and restarting it i see preview js log window, which complains about npm not being at version 7. I will try that.

**Comment by fwouts** [comment, marked as answer]
For future reference, I've added some documentation: https://previewjs.com/docs/platforms/jetbrains
